sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-process-integration.scheduler.models.ts

```diff
@@ -56,7 +56,7 @@ export const MSFAA_PART_TIME_OTHER_COUNTRY: MSFAATestInputData = {
   birthDate: "2000-01-01",
   surname: "Other Doe",
   givenName: "Jane",
-  gender: "unknown",
+  gender: "nonBinary",
```

Comment:
Just to keep the equivalency, should it not be "Prefer not to answer"?

Reply:
updated